web/src/pages/barcoding.js

```diff
@@ -1,9 +1,124 @@
-import React from 'react'
+import React from 'react';
+import tw from 'twin.macro';
+
+import styled from 'styled-components';
+
+import subway from '../images/barcoding/subway-takeover.png';
+import mask from '../images/barcoding/Mask-Group.png';
+import welcome from '../images/barcoding/Welcome-Box-2.png';
+import billboard from '../images/barcoding/Billboard-mockup-1.png';
+import masktwo from '../images/barcoding/Mask-Group(1).png';
+import office from '../images/barcoding/Office.png';
+import signage from '../images/barcoding/Signage.png';
+import pattern from '../images/barcoding/Pattern.png';
+import lockup from '../images/barcoding/Barcoding-LockUp.png';
+import video from '../images/barcoding/Video-pic.png';
+import postcard1 from '../images/barcoding/Postcard-BG1.png';
+import postcard2 from '../images/barcoding/Postcard-BG2.png';
+import document from '../images/barcoding/B-document.png';
+import exploration from '../images/barcoding/B-Exploration.jpg';
+import tape from '../images/barcoding/tape-pattern.png';
+
+import Layout from '../components/Layout'
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
```

Comment:
I think you can remove both width declarations, they don't seem to be doing anything. Also the top: -160px isn't doing anything and the hero image is behind the header on mobile. Should change to `padding-top: 160px`